src/main/kotlin/io/github/mojira/arisa/domain/ChangeLogItem.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.mojira.arisa.domain
+
+data class ChangeLogItem(
+    val created: Long,
```

Comment:
Probably for Long too, as Instant provides an easy function to obtain millis anyways. It just helps with what you should expect when seeing created / updated.

Reply:
I would argue that we should use Instant everywhere

Reply:
I actually already changed it to Instant when fixing build errors

Reply:
https://github.com/mojira/arisa-kt/issues/206